docker/flags.go

```diff
@@ -50,7 +50,7 @@ var (
 		{"logs", "Fetch the logs of a container"},
 		{"port", "Lookup the public-facing port that is NAT-ed to PRIVATE_PORT"},
 		{"pause", "Pause all processes within a container"},
-		{"ps", "List containers"},
+		{"ps", "List running containers"},
```

Comment:
Yes, I think `List containers` is correct here.  Your other changes looks good if you want to remove this one. Thanks!